docs/components/connectors/introduction.md

```diff
@@ -2,125 +2,45 @@
 id: introduction-to-connectors
 title: Introduction
 description: "A Connector is a reusable building block that performs the integration with an external system and works out of the box."
+keywords: [bridge, reusable component, low code]
 ---
 
-A **Connector** is a reusable building block that performs the integration with an external system and works out of the box.
+A **Connector** is a reusable building block that performs the integration with an external system.
 
-<!-- taken from https://docs.google.com/presentation/d/1F_xr6LSmut84up_OkyEwgGh9JSkQuhyqmTihM0TlavE/edit#slide=id.g15a5b1f60a9_0_3408 -->
+Most processes require interaction with systems outside of Camunda to orchestrate and complete work.
 
-![connectors concept](img/connector-concept.png)
+- In **human task orchestration**, actors are notified of pending or completed processes through communication tools such as [Slack](/components/connectors/out-of-the-box-connectors/outbound/slack.md) or [Microsoft Teams](/components/connectors/out-of-the-box-connectors/outbound/microsoft-teams.md) for example in onboarding flow.
+- In **business processes**, records are created or updated in enterprise business systems.
```

Comment:
I purposefully omitted any examples from the bullet point on business processes, since we don't have an OOTB SAP or Salesforce Connector yet.